nav2_controller/include/nav2_controller/controller_server.hpp

```diff
@@ -176,6 +176,10 @@ class ControllerServer : public nav2_util::LifecycleNode
    * @brief Calls velocity publisher to publish zero velocity
    */
   void publishZeroVelocity();
+  /**
```

Comment:
Space between dogyxen and last method

Reply:
This is not consistent throughout the header file. I can add this if you want.